modules/redux/test/integration.test.ts

```diff
@@ -0,0 +1,252 @@
+import http from "http";
+import { it, describe, before, after } from "mocha";
+import { createStore, combineReducers, applyMiddleware } from "redux";
+import PhenylHttpServer from "../../http-server";
+import PhenylRestApi from "../../rest-api";
+import PhenylHttpClient from "../../http-client";
+import {
+  GeneralTypeMap,
+  KvsClient,
+  Session,
+  PhenylAction,
+  LocalState,
+  GetCommandResult
+} from "../..//interfaces";
+import { createEntityClient } from "../..//memory-db";
+import { StandardUserDefinition } from "../../standards";
+import assert from "assert";
+
+import { PhenylRedux, LocalStateFinder } from "../src";
+import { PhenylReduxModule } from "../src/phenyl-redux-module";
+
+type PlainPatient = {
+  id: string;
+  name: string;
+  email: string;
+  password: string;
+};
+
+type PatientRequest = {
+  id: string;
+  name: string;
+  email: string;
+  password: string;
+};
+
+type PatientResponse = {
+  id: string;
+  name: string;
+  email: string;
+};
+
+type Credentials = {
+  email: string;
+  password: string;
+};
+
+type MyAuthSetting = {
+  credentials: Credentials;
+  options: {};
+};
+
+type MyEntityMap = {
+  patient: PlainPatient;
+};
+type MyGeneralReqResEntityMap = {
+  patient: {
```

Comment:
[imo] utilize ReqRes
```suggestion
  patient: ReqRes<PatientRequest, PatientResponse>
```